socorro/external/es/mappings/correlations_index_settings.json

```diff
@@ -0,0 +1,65 @@
+{
+    "mappings": {
+        "correlations": {
+            "_all": {
+                "enabled": false
+            },
+            "_source": {
+                "compress": true
+            },
+            "properties": {
+                "product": {
+                    "type": "string",
```

Comment:
Why isn't this `analyzer: keyword` too? Is it out realistic fear that we might have corrections for "Firefox" AND "firefox"?